contracts/interfaces/IStakingPool.sol

```diff
@@ -33,13 +35,28 @@ interface IStakingPool is IERC721 {
   // stakers are grouped based on the timelock expiration
   // group index is calculated based on the expiration date
   // the initial proposal is to have 4 groups per year (1 group per quarter)
-  struct StakeGroup {
+  struct Group {
```

Comment:
I like the name Term, it's specific to time intervals, while the Group is a name for a collection of similar things

Reply:
> Group is a name for a collection of similar things

That's the exact intention. A group of stakers.

> Term?
It may have more sense when thinking about it from a user's point of view so you could use that language in the UI.
In the contract itself it won't be immediatelly clear why a struct named `Term` would have stake and reward shares, while it's easier to understand why a group of stakers would have those.

Reply:
Virtually all definitions of the word "term" represent an amount/period of time, not an expiry date. In our case the "term" is 91 days (GROUP_DURATION).

https://www.investopedia.com/terms/t/term.asp

Reply:
See my link from invetopedia. ^